layers/+fun/emoji/packages.el

```diff
@@ -39,4 +39,21 @@
     (use-package company-emoji
       :if (configuration-layer/package-usedp 'company)
       :defer t
-      :init (setq company-emoji-insert-unicode nil))))
+      :init
+      (progn
+        (setq company-emoji-insert-unicode nil)
+        ;; From https://github.com/dunn/company-emoji/README.md for Linux, or on Mac OS X and using the Cocoa version of Emacs
+        (defun --set-emoji-font (frame)
```

Comment:
Wasn't there a more useful function than display-graphic-p? so that this also works when spawned in daemon mode? @travisbhartwell